lib/reverse_words.rb

```diff
@@ -1,6 +1,37 @@
 # A method to reverse each word in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: The time complexity is O(n^2) where n is the length of my_words
```

Comment:
I believe the time complexity would be linear O(n). O(n<sup>2</sup>) occurs when you have nested `for` loops. Since you are not nesting your two loops, I do not think it's O(n<sup>2</sup>). Spoiler alert: I could totally be wrong too because the amount of times I have had to worry about Big O at my job is zero. 😄 

This document helped me a lot with Big O when I was in Ada: https://rob-bell.net/2009/06/a-beginners-guide-to-big-o-notation/